releases/2.13.9.md

```diff
@@ -0,0 +1,78 @@
+* Tasty Reader: Restrict access to experimental definitions ([#10020](https://github.com/scala/scala/pull/10020))
+* To aid cross-building, accept and ignore `using` in method calls ([#10064](https://github.com/scala/scala/pull/10064) by [@som-snytt](https://github.com/som-snytt))
+* To aid cross-building, allow `?` as a wildcard even without `-Xsource:3` ([#9990](https://github.com/scala/scala/pull/9990))
+* Make Scala-3-style implicit resolution explicitly opt-in rather than bundled in `-Xsource:3` ([#10012](https://github.com/scala/scala/pull/10012) by [@povder](https://github.com/povder))
+* Prefer type of overridden member when inferring (under `-Xsource:3`) ([#9891](https://github.com/scala/scala/pull/9891) by [@som-snytt](https://github.com/som-snytt))
+
+### JDK version support
+
+* Make `-release` more useful, deprecate `-target`, align with Scala 3 ([#9982](https://github.com/scala/scala/pull/9982) by [@som-snytt](https://github.com/som-snytt))
+* Support JDK 19 ([#10001](https://github.com/scala/scala/pull/10001) by [@Philippus](https://github.com/Philippus))
+
+### Warnings and lints
+
+* Add `-Wnonunit-statement` to warn about discarded values in statement position ([#9893](https://github.com/scala/scala/pull/9893) by [@som-snytt](https://github.com/som-snytt))
+* Make unused-import warnings easier to silence (support filtering by `origin=`) ([#9939](https://github.com/scala/scala/pull/9939) by [@som-snytt](https://github.com/som-snytt))
+* Add `-Wperformance` lints for `*Ref` boxing and nonlocal `return` ([#9889](https://github.com/scala/scala/pull/9889) by [@som-snytt](https://github.com/som-snytt))
+
+### Language improvements
+
+* Improve support for Unicode supplementary characters in identifiers and string interpolation ([#9805](https://github.com/scala/scala/pull/9805) by [@som-snytt](https://github.com/som-snytt))
+
+### Compiler options
+
+* Use subcolon args to simplify optimizer options ([#9810](https://github.com/scala/scala/pull/9810) by [@som-snytt](https://github.com/som-snytt))
+* For troubleshooting compiler, add `-Vdebug-type-error` (and remove `-Yissue-debug`) ([#9824](https://github.com/scala/scala/pull/9824) by [@tribbloid](https://github.com/tribbloid))
+
+### Security
+
+* Error on source files with Unicode directional formatting characters ([#10017](https://github.com/scala/scala/pull/10017))
+* Prevent `Function0` execution during `LazyList` deserialization ([#10118](https://github.com/scala/scala/pull/10118))
+
+### Bugfixes
+
+* Emit all bridge methods non-final (perhaps affecting serialization compat) ([#9976](https://github.com/scala/scala/pull/9976))
+* Fix null-pointer regression in `Vector#prependedAll` and `Vector#appendedAll` ([#9983](https://github.com/scala/scala/pull/9983))
+* Improve concurrent behavior of Java `ConcurrentMap` wrapper
+ ([#10027](https://github.com/scala/scala/pull/10027) by [@som-snytt](https://github.com/som-snytt))
```

Comment:
please substitute

Igor Gabaydulin <igabaydulin@gmail.com> or https://github.com/igabaydulin

who was the real instigator.

Reply:
will do